.cypress/integration/notebooks_test/notebooks.spec.js

```diff
@@ -66,157 +68,126 @@ describe('Adding sample data and visualization', () => {
-
-    // keep a notebook for testing
-    cy.get('.euiButton__text').contains('Create notebook').click();
-    cy.get('input.euiFieldText').type(TEST_NOTEBOOK);
-    cy.get('.euiButton__text')
-      .contains(/^Create$/)
-      .click();
+  it('Deletes all notebooks', () => {
+    cy.get('input[data-test-subj="checkboxSelectAll"]').click();
+    cy.get('button[data-test-subj="notebookTableActionBtn"]').click();
+    cy.get('button[data-test-subj="deleteNotebookBtn"]').click();
+    cy.get('button[data-test-subj="delete-notebook-modal-delete-button"]').should('be.disabled');
+    cy.get('input[data-test-subj="delete-notebook-modal-input"]').focus().type('delete');
+    cy.get('button[data-test-subj="delete-notebook-modal-delete-button"]').should(
+      'not.be.disabled'
+    );
+    cy.get('button[data-test-subj="delete-notebook-modal-delete-button"]').click();
+    moveToNotebookHome();
+    cy.get('div[data-test-subj="notebookEmptyTableText"]').should('exist');
   });
 });
 
-describe('Test reporting integration if plugin installed', () => {
-  beforeEach(() => {
-    cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-notebooks#/`);
-    cy.get('.euiTableCellContent').contains(TEST_NOTEBOOK).click();
-    cy.wait(delay); //page needs to process before checking
-    cy.get('body').then(($body) => {
-      skipOn($body.find('#reportingActionsButton').length <= 0);
-    });
-  });
-
-  it('Create in-context PDF report from notebook', () => {
-    cy.get('#reportingActionsButton').click();
-    cy.get('button.euiContextMenuItem:nth-child(1)').contains('Download PDF').click();
-    cy.get('#downloadInProgressLoadingModal').should('exist');
-  });
-
-  it('Create in-context PNG report from notebook', () => {
-    cy.get('#reportingActionsButton').click();
-    cy.get('button.euiContextMenuItem:nth-child(2)').contains('Download PNG').click();
-    cy.get('#downloadInProgressLoadingModal').should('exist');
-  });
-
-  it('Create on-demand report definition from context menu', () => {
-    cy.get('#reportingActionsButton').click();
-    cy.get('button.euiContextMenuItem:nth-child(3)').contains('Create report definition').click();
-    cy.location('pathname', { timeout: 60000 }).should('include', '/reports-dashboards');
-    cy.get('#reportSettingsName').type('Create notebook on-demand report');
-    cy.get('#createNewReportDefinition').click({ force: true });
-  });
-
-  it('View reports homepage from context menu', () => {
-    cy.get('#reportingActionsButton').click();
-    cy.get('button.euiContextMenuItem:nth-child(4)').contains('View reports').click();
-    cy.location('pathname', { timeout: 60000 }).should('include', '/reports-dashboards');
+describe('Testing paragraphs', () => {
+  before(() => {
+    moveToNotebookHome();
+    cy.get('a[data-test-subj="createNotebookPrimaryBtn"]').click();
+    cy.get('input[data-test-subj="custom-input-modal-input"]').focus().type(TEST_NOTEBOOK);
```

Comment:
In the cypress documentation it says "It is unsafe to chain further commands that rely on the subject after .focus()" https://docs.cypress.io/api/commands/focus 
It may not cause any issues but its worth keeping an eye on.

Reply:
Make sense, this wasn't flaky when I tested it out on local with 20 runs. Without focus being called the type wouldn't work as expected. Will leave this one as is, let me know if you think this should be updated before merging in the PR.

Reply:
IMO it doesn't need to be updated if it isn't causing issues, but it's definitely something to look at if further issues crop up.